### ISSUE TITLE
Tag2Pix: Line Art Colorization Using Text Tag With SECat and Changing Loss

### ISSUE THREAD

**Issue by SerialLain3170** [SerialLain3170/nagato#6]
# Overview
![image](https://user-images.githubusercontent.com/32360147/99740855-27f9fe00-2b13-11eb-88c0-ae42f5dae98c.png)
- This method aims to colorize taking the line arts and tags as the input.
- Authors apply various line extraction methods in order not to avoid overfitting to single type of line art.
- There are mainly three contributions to colorization from tags.
  - In addition to discriminating if the output is real or fake, discrimintor classifies color variant tags (CVT) and color invariant tags (CIT).
  - In order to give encoded features of color variant tags, authors proposed SECat (Squeeze-Exciration Concatanation) module inspired by StyleGAN.
![image](https://user-images.githubusercontent.com/32360147/99741787-67c1e500-2b15-11eb-8257-832389fd4bfc.png)
  - Authors design loss functions with two step curriculum training.
        - 1st step: this step calculates adversarial loss and reconstruction loss only. The weight of reconstruction loss is larget enough to cover the original color distribution.
        - 2nd step: aside from adversrial loss and recontruction loss, this step calculates classification loss.

**Comment by SerialLain3170** [comment]
# Evaluation
![image](https://user-images.githubusercontent.com/32360147/99742738-7d380e80-2b17-11eb-83a9-ce12a9065f48.png)
- As for the comparions between sketch-based network, authors compared Tag2Pix with PaintsChainer (color stroke-based) and Style2paints (reference-based). 

![image](https://user-images.githubusercontent.com/32360147/99743046-2bdc4f00-2b18-11eb-9dd7-0a89bcacdd30.png)
- When it comes to the comparions between tag-based network, authors compared Tag2Pix with sentence-based method (Manjunatha et al.). 

![image](https://user-images.githubusercontent.com/32360147/99743498-229fb200-2b19-11eb-947d-baa53e3283b7.png)
![image](https://user-images.githubusercontent.com/32360147/99743514-2d5a4700-2b19-11eb-9492-9ccaca185e08.png)
- Finally, authors compared among CVT embedding schemes. They compared SECat with various methods from naive ResNeXt block to AdaIN. With the aspect of FID and computational complexity, SECat attained the lowest FID while keeping the reasonable amout of parameters.

**Comment by SerialLain3170** [comment]
# Supplemental information
[Paper Link](https://arxiv.org/pdf/1908.05840.pdf)

### Authors
Hyunsu Kim, Ho Young Jhoo, Eunhyeok Park, and Sungjoo Yoo

### Organization
Seoul National University

### Conference
ICCV 2019
